test_prebuild/test_track_variables.py

```diff
@@ -0,0 +1,122 @@
+#! /usr/bin/env python3
+"""
+-----------------------------------------------------------------------
+ Description:  Contains unit tests for ccpp_track_variables.py script
+
+ Assumptions:
+
+ Command line arguments: none
+
+ Usage: python test_var_transforms.py         # run the unit tests
+-----------------------------------------------------------------------
+"""
+import sys
+import os
+import pytest
+
+TEST_DIR = os.path.dirname(os.path.abspath(__file__))
+SCRIPTS_DIR = os.path.abspath(os.path.join(TEST_DIR, os.pardir, "scripts"))
+SAMPLE_FILES_DIR = "test_track_variables"
+SUITE_FILE = f'{SAMPLE_FILES_DIR}/suite_TEST_SUITE.xml'
+SMALL_SUITE_FILE = f'{SAMPLE_FILES_DIR}/suite_small_suite.xml'
+CONFIG_FILE = f'{SAMPLE_FILES_DIR}/ccpp_prebuild_config.py'
+if not os.path.exists(SCRIPTS_DIR):
+    raise ImportError(f"Cannot find scripts directory {SCRIPTS_DIR}")
+
+sys.path.append(SCRIPTS_DIR)
+
+from ccpp_track_variables import track_variables
+
+def test_successful_match(capsys):
+    expected_output = """For suite test_track_variables/suite_small_suite.xml, the following schemes (in order for each group) use the variable air_pressure:
```

Comment:
A doc string explaining the purpose of the test (what is being tested, test limits, etc.) would be helpful here and for all the test functions below.

Reply:
Agreed